api/proto/dsse.proto

```diff
@@ -16,16 +16,25 @@ syntax = "proto3";
 package dev.sigstore.rekor.v2;
 
 import "google/api/field_behavior.proto";
+import "sigstore_common.proto";
+import "envelope.proto";
 
 import "verifier.proto";
 
 option go_package = "github.com/sigstore/rekor-tiles/pkg/generated/protobuf";
 
 // A request to add a dsse entry to the log
-message DSSERequest {
-    // A DSSE envelope (as defined by https://github.com/secure-systems-lab/dsse/blob/master/envelope.proto)
-    // in stringified JSON
-    string envelope = 1 [(google.api.field_behavior) = REQUIRED];
+message DSSERequestV0_0_2 {
+    // A DSSE envelope
+    io.intoto.Envelope envelope = 1 [(google.api.field_behavior) = REQUIRED];
     // collection of verification material used to verify
-    repeated Verifier verifier = 2;
+    repeated Verifier verifiers = 2 [(google.api.field_behavior) = REQUIRED];
 }
+
+
+message DSSELogEntryV0_0_2 {
+    // The hash of the dsse payload
+    dev.sigstore.common.v1.HashOutput payloadHash = 2 [(google.api.field_behavior) = REQUIRED];
+    // A collection of all signatures over the payload
+    repeated bytes signatures = 3 [(google.api.field_behavior) = REQUIRED];
```

Comment:
I think this needs the verifier too. The old DSSE had that in the persisted object https://github.com/sigstore/rekor/blob/73dba7c07d0747f00119417fc0ff994a393f97b2/pkg/types/dsse/v0.0.1/entry.go#L235-L238